persistence/src/main/scala/com/mesosphere/usi/repository/PodRecordRepository.scala

```diff
@@ -0,0 +1,26 @@
+package com.mesosphere.usi.repository
+
+import com.mesosphere.usi.core.models.{PodId, PodRecord}
+
+import scala.concurrent.Future
+
+/**
+  * Repository that stores [[PodRecord]].
+  */
+trait PodRecordRepository {
+
+  /**
+    * Stores the provided [[PodRecord]] in the repository.
+    * @param record
+    * @return
+    */
+  def store(record: PodRecord): Future[Unit]
```

Comment:
How about all four CRUD methods for the time being? And also, how about staying with "default names" e.g. `create`, `read` etc.?

Reply:
Yeah, why not use @zen-dog's storage?

Reply:
@jeschkies  it is the next step, @zen-dog is operating on a lower level